src/foundry/client/data/documents/token.d.mts

```diff
@@ -861,28 +862,49 @@ declare global {
         >
       | ActorDelta.OnDeleteDescendantDocumentsArgs;
 
-    /**
-     * @deprecated {@link TokenDocument.Database | `TokenDocument.DatabaseOperation`}
-     */
-    // eslint-disable-next-line @typescript-eslint/no-deprecated
-    interface DatabaseOperations extends Document.Database.Operations<TokenDocument.Implementation> {}
+    // The getBarAttribute monkeypatch is simply inside the data model definition at `src\foundry\common\data\data.d.mts`
```

Comment:
Nit; I'd prefer Linux paths, if only because that's how most paths have been written thus far.